### PR TITLE
sending cli error message in send events and fixing stdin bug

### PR DESCRIPTION
**Pull request by prklm10**
sending cli error message in send events and fixing stdin bug
- Not sending cli error when `PERCY_CLIENT_ERROR_LOGS` is set to `false`.
- Otherwise sending error to send events.
- Fixing stdin bug introduced  in `1.27.4`.

### CODE REVIEW

File: packages/cli-exec/src/exec.js [hunk 1]
```diff
@@ -121,13 +125,17 @@ async function* spawn(cmd, args, percy) {
     proc.on('close', code => {
       closed = code;
       if (code !== 0) {
+        // Only send cli error when PERCY_CLIENT_ERROR_LOGS is set to true
+        if (process.env.PERCY_CLIENT_ERROR_LOGS && process.env.PERCY_CLIENT_ERROR_LOGS === 'false') {
```
Comment (ninadbstack) [inline packages/cli-exec/src/exec.js:129]:
```suggestion
        if (process.env.PERCY_CLIENT_ERROR_LOGS === 'false') {
```

File: packages/cli-exec/src/exec.js [hunk 2]
```diff
@@ -121,13 +125,17 @@ async function* spawn(cmd, args, percy) {
...
+          errorMessage = '1';
```
Comment (ninadbstack) [inline packages/cli-exec/src/exec.js:130]:
```suggestion
          errorMessage = 'Log sharing is disabled';
```

File: packages/cli-exec/test/test_prog.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+// Error
```
Comment (ninadbstack) [inline packages/cli-exec/test/test_prog.js:1]: move this to test/test_data folder

File: packages/cli-exec/src/exec.js (hunk 1 above)
Comment (prklm10) [reply packages/cli-exec/src/exec.js:129]: updated.

File: packages/cli-exec/src/exec.js (hunk 2 above)
Comment (prklm10) [reply packages/cli-exec/src/exec.js:130]: Updated

File: packages/cli-exec/test/test_prog.js (hunk 3 above)
Comment (prklm10) [reply packages/cli-exec/test/test_prog.js:1]: done

**Pull request merged**